pkgs/shells/nushell/default.nix

```diff
@@ -55,7 +55,7 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://www.nushell.sh/";
     license = licenses.mit;
     maintainers = with maintainers; [ Br1ght0ne johntitor marsam ];
-    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "aarch64-linux" ];
+    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];
```

Comment:
I think we can just drop `platforms` here because it should build on all. In this case it defaults to `rustc.meta.platforms`.

Reply:
```suggestion
```

Reply:
Addressed in 459fbe19317baabd332f9000c1f0eb1a2a6bac99